crates/config/src/revive.rs

```diff
@@ -11,14 +14,36 @@ pub const REVIVE_ARTIFACTS_DIR: &str = "revive-out";
 
 pub const CONTRACT_SIZE_LIMIT: usize = 250_000;
 
+/// Variants for selecting the [`Revive`] instance
```

Comment:
why do we need the ReviveReq - is there anything missing in `SolcReq`?

Reply:
we want to have `semver::VersionReq` instead of plain `Version`

Reply:
dunno honestly i usually specify semver compiler version when i use other tooling, so i though that the common convention probably can apply here. However i can revert this change to use `solcReq` back

Reply:
I prefer to keep the `SolcReq` if it is not a big issue

Reply:
done